service-workers/cache-storage/script-tests/cache-storage.js

```diff
@@ -170,8 +170,15 @@ promise_test(function(t) {
 
 promise_test(function(t) {
     var bad_name = 'unpaired\uD800';
-    var converted_name = 'unpaired\uFFFD'; // Don't create cache with this name.
-    return self.caches.has(converted_name)
+    var converted_name = 'unpaired\uFFFD';
+
+    return Promise.all([
+          self.caches.delete(bad_name),
+          self.caches.delete(converted_name)
```

Comment:
Quoting from above:

> One test asserts that the cache name "unpaired\uD800" does not get mapped to "unpaired\uFFFD", but relies on a cache by that name not existing. If an implementation is buggy it will implicitly create the cache, causing the test to fail the setup checks on subsequent runs, as opposed to hitting the actual test assertions.
>
> Change the test to delete the caches so that it starts with a clean slate regardless of whether or it passes the test.